test/parallel/test-async-wrap-uid.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+require('../common');
+const fs = require('fs');
+const assert = require('assert');
+const async_wrap = process.binding('async_wrap');
```

Comment:
I know, but this is consistent with the other async wrap tests https://github.com/nodejs/node/blob/master/test/parallel/test-async-wrap-propagate-parent.js#L6 and also some `child_process` tests https://github.com/nodejs/node/blob/master/test/parallel/test-child-process-fork3.js